src/Application/Game.cpp

```diff
@@ -1005,7 +1005,8 @@ void Game::EventLoop() {
                 case UIMSG_TransitionUI_Confirm:
                     pMessageQueue_50CBD0->Flush();
                     dword_50CDC8 = 1;
-                    pAudioPlayer->PlaySound(SOUND_StartMainChoice02, PID_INVALID, 0, -1, 0, 0);
+                    // PID was PID_INVALID
+                    pAudioPlayer->PlaySound(SOUND_StartMainChoice02, -1, 0, -1, 0, 0);
```

Comment:
We should probably add enums to cover cases where we play sounds but not using PID. Would reduce magic numbers. SOUND_ORIGIN_PARTY ect or something. Please add a TODO in PlaySound.

Reply:
Replacing `PID_INVALID` with `-1` is a bad idea.

Imagine we wanted to refactor pids and properly encapsulate them in `class Pid`. Note that this refactoring is not something out of SF, and it will happen.

So now you're using a number here that cannot be represented as a value of the newly introduced `class Pid`. What do we replace it with then? `std::optional<Pid>`?

Long story short, don't do it. An OK fix, for now, would be to assert in `Play` that the passed pid is not `-1` and fix all call sites to pass `PID_INVALID` instead.

And yes, let's add a `TODO` for a name that actually conveys the semantics.